scripts/postProcessTypes.js

```diff
@@ -7,3 +7,19 @@ replace.sync({
   from: [/agIdSequence/g, /caIdSequence/g, /caDocLink/g],
   to: ['aGIdSequence', 'cAIdSequence', 'cADocLink'],
 });
+
+// Add in import for modified schema
+replace.sync({
+  files: path.resolve(definitionsDir, 'augment-api-rpc.ts'),
+  from: ['\n\n'], // there should be two new lines between the comment and import block to target
```

Comment:
This seems brittle. Can't you do something like `\nimport` and then replace it with the same thing you're replacing with now but add the `import` at the end?